tutorials/34_Model_Based_Evaluation_of_RAG_Pipelines.ipynb

```diff
@@ -0,0 +1,4480 @@
+{
```

Comment:
<p>How can I learn other <code>llm_persona</code> options? Is it something like a system prompt?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
I added a sentence explaining llm_persona. And yes, it will be included in the prompt.